nss-day-cohort-52/giffygram-wild-hurricanes
# HTML template

#### Issues Completed
Applies to issue #37 

#### Changes Made
1.  [x] Feature
    [] Bug-fix
    [] Feature that will cause a bug
    [] Not a feature or a bug- requires documentation
2. Added files MessageList, Post, PostList, PrivateMessages, Footer, NavBar
3. Modified the above module and the GiffyGram module with HTML rendering functions.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    commit your current code (git add --all, git commit -m "")
    git fetch --all
    git checkout branchname
    ```
2. Serve the code to browser.
3. Test app functionality.
    > How should the reviewer test functionality? serve
    > What is the intended outcome? all portions visible except message list
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
